aqueduct_client/aqueduct_client.py

```diff
@@ -262,6 +272,27 @@ def get_pipeline_results_dataframe(self, execution_id):
 
         return result_df
 
+    def submit_then_get_pipeline_execution(self,
```

Comment:
With the addition of this function the overall verbiage of the functions starts to read a little awkwardly.
At least in our use case, our main usage would be this `submit_then_get_pipeline_execution`; which I could see being the "main" entry point into the client for others as well.

Should this be `run_pipeline` to match https://www.quantopian.com/help#quantopian_research_run_pipeline ?

Reply:
I was trying to keep this inline with the existing aq verbiage where you `submit` and then later `get`, but happy to change this to anything.